ArchUnitNET/Loader/ArchBuilder.cs

```diff
@@ -43,7 +44,7 @@ public ArchBuilder()
             _architectureCache = ArchitectureCache.Instance;
         }
 
-        public IEnumerable<IType> Types => _architectureTypes;
+        public IEnumerable<IType> Types => _architectureTypes.Select(pair => pair.Value);
```

Comment:
Spricht etwas dagegen hier die Values-Property für Dictionaries zu verwenden? Fände ich noch einfacher lesbar. :)
```suggestion
        public IEnumerable<IType> Types => _architectureTypes.Values;
```